### DISCUSSION TITLE
Is it possible to maintain the folder structure in the repo?

### DISCUSSION THREAD

**Discussion by smokeyfro** [ObsidianPublisher/obsidian-github-publisher#307] [Q&A — Plugin]
Hello. I recently started on a quest to move my Astro powered sites content into a symlinked git submodule, with the intent of using Obsidian to author my sites content. I just did a test run and while it's published all the items to the repo, they are all nested in a new /src/content folder without any subdirectories.

This is my current folder setup:

.sf-content/changelogs/
.sf-content/journal/
.sf-content/pages/
.sf-content/themes/

... etc

The folder structure is maintained inside Obsidian, but I can't figure out how to sync the content repo maintaining the same folder structure.

I have Share all files enabled.

Any ideas would be much appreciated :)

TIA,
Chris

**Comment by Mara-Li** [comment]
You need to open the settings and check the path (second tab). You can change here. I didn't understand what setup do you want, but you can send file into a specific folder, maintaining Obsidian structure, and a frontmatter key. 
If I understand well, you need to check the "Obsidian" for the tree.
